db/schema.rb

```diff
@@ -427,6 +427,23 @@
     t.index ["legal_aid_application_id"], name: "index_dependants_on_legal_aid_application_id"
   end
 
+  create_table "discretionary_disregards", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
```

Comment:
question: are we happy with this as the data model for discretionary_disregards given the work on ap-5349 to come?

Reply:
given that it follows the naming convention for policy_disregards table and model I think this is ok, but might be better named as `discretionary_policy_disregards`, to separate this from "disregarded benefits" naming. 

Though I do not have strong feelings on these I mention them here for completeness

- we could rename the policy_disregards table as mandatory_policy_disregards if that naming is logical (I am not sure)
- its a shame we do not have a table structure that allows us to extend the existing table more easily by having a `legal_aid_application_policy_disregards` join table  and  a `policy_disregards` table that is seed with records with `name` and `mandatory:boolean` or similar to achieve this functionality more elegantly - like the `transaction_types` table and equivalent `legal_aid_application_transaction_types` join table. Perhaps we should have a tech debt ticket 🤔 ?!